compose/src/commonMain/kotlin/Kanvas.kt

```diff
@@ -0,0 +1,18 @@
+package com.juul.krayon.compose
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Path
+import com.juul.krayon.kanvas.Kanvas
+
+@Composable
+public fun Kanvas(
+    modifier: Modifier,
+    onDraw: Kanvas<Path>.() -> Unit,
+) {
+    val resourceContext = rememberResourceCache()
```

Comment:
I a bit confused. Variable name is `resourceContext` but function talks about "cache"?
Are we just caching the `Context` here?

Reply:
Missed a rename, 😅. This class started off as just containing Android's `Context`, but then I decided to implement the `PaintCache` because it seemed like a low effort efficiency win.